desktop/libs/hadoop/src/hadoop/fs/upload.py

```diff
@@ -221,6 +221,41 @@ def close(self):
     self._file.close()
 
 
+class CustomDelegatingUploadHandler(FileUploadHandler):
+  """
+  Delegates the upload handling based on the request URL.
+
+  When the request URL starts with "/desktop/api2/doc/import" (indicating a document
+  import), delegate all processing to HDFSfileUploadHandler.
+  Otherwise, delegate to FineUploaderChunkedUploadHandler.
+  """
+
+  def __init__(self, request, *args, **kwargs):
+    super().__init__(request, *args, **kwargs)
+    if request.path.startswith("/desktop/api2/doc/import"):
```

Comment:
Try using `reverse()` instead of hardcoding here?

Reply:
Made use of reverse() instead of hardcoding the path here.